src/components/GTFOBins/GtfoMain.jsx

```diff
@@ -0,0 +1,127 @@
+import { FunctionData } from "./_gtfoJson";
+
+export default function GtfoBin() {
+    const [activeFunction, setActiveFunction] = useState("All");
+
+    useEffect(() => {}, [activeFunction]);
+
+    function handleActiveFunction(data) {
+        setActiveFunction(data);
+    }
+
+    return (
+        <Wrapper style={{ width: "60%" }}>
+            <label style={{ fontSize: "48px", fontWeight: "bold", letterSpacing: "5px" }}>BinExploit</label>
+
+            <div style={{ display: "block", width: "100%", textAlign: "center", margin: "30px" }}>
+                <button
+                    style={{ padding: "10px", border: "solid 1px", margin: "10px" }}
+                    onClick={() => {
+                        handleActiveFunction("All");
+                    }}
+                >
+                    All
+                </button>
+
+                {Object.keys(FunctionData).map((data) => (
+                    <button
+                        style={{ padding: "10px", border: "solid 1px", margin: "10px" }}
+                        key={data}
+                        onClick={() => {
+                            handleActiveFunction(data);
+                        }}
+                    >
+                        {data}
+                    </button>
+                ))}
+            </div>
+            <div
+                style={{
+                    display: "flex",
+                    flexDirection: "column",
+                    justifyContent: "center",
+                    alignContent: "center",
+                    padding: "30px",
+                }}
+            >
+                {/* <Search
+                    id="Search"
+                    placeholder={"Search among " + Object.keys(Data[0]).length + " Binaries"}
+                    style={{ marginBottom: "20px", marginLeft: "0" }}
+                /> */}
+                {Data.map((data, index) =>
+                    Object.keys(data).map((keys) => {
+                        if (activeFunction === "All") {
+                            return (
+                                <div style={{ display: "flex" }} key={keys}>
+                                    <a
+                                        style={{
+                                            width: "180px",
+                                            color: "white",
+                                            display: "flex",
+                                            alignItems: "center",
+                                            textDecorationLine: "underline",
+                                        }}
+                                        href={"/tools/binexploits/" + keys}
+                                    >
+                                        {keys}
+                                    </a>
+                                    <div style={{ display: "grid-row", width: "550px" }}>
+                                        {Object.keys(data[keys].functions).map((function_) => (
```

Comment:
what do you say about `binaryFunction` instead of `function_`? or even "func" as you did later on